src/engraving/rw/scorereader.cpp

```diff
@@ -140,6 +140,10 @@ Err ScoreReader::loadMscz(Ms::MasterScore* masterScore, const mu::engraving::Msc
         }
     }
 
+    // These are necessary to be sure that ALL the layout is correctly updated when loading any file
+    masterScore->styleChanged();
+    masterScore->update();
+
```

Comment:
@mike-spa can we remove this block of code?
This code takes a lot of time when opening a big score.
You can test on the score from this PR https://github.com/musescore/MuseScore/issues/8143

This code does not solve all the problems with the long opening, but it significantly reduces the time.
And the comment says that this code may not be executed, we just make sure.

Reply:
@Eism I had to do that because many scores were not loading the new spacing settings correctly, so for now we can't just remove the code, unfortunately. But there is probably a smarter way to do it, I can try to look into that. Just for my understanding: which one of them is more time consuming? `styleChanged()`or `update()`?

Side note: #8143 was opened before this PR was merged, so there must be also something else going on, I guess.

Reply:
Score::update

![image](https://user-images.githubusercontent.com/10116828/164615083-a4242bf2-0afd-4add-aaf8-433748e37ba1.png)

Reply:
Excellent, thank you. I'll try to look into it as soon as I can!